ansible-scylla-node/defaults/main.yml

```diff
@@ -291,6 +291,36 @@ data_encryption_local:
   secret_key_block_mode: CBC # mandatory if you're not providing your own key
   secret_key_padding: PKCS5 # mandatory if you're not providing your own key
 
+# Encrypt system resources:
+# System encryption is applied to semi-transient on-disk data, such as commit logs, batch logs, and hinted handoff data.
+# If you'd like to use a LocalKeyProvider, please use the system_info_encrypt_local config
+# If you'd like to use a KMIPKeyProvider, please use the system_info_encryption_kmip config
+
+# LocalKeyProvider:
+# To use your own key, copy it to {{ localhost_key_dir }} with the name {{ key_name }} and then this key will be copied to all the nodes.
+# If you don't provide your own key, the role will automatically generate a new one for you based on the variables under system_info_encryption_local and at
+# the end of the process the role will create {{ localhost_key_dir }} (if it's not already created) and copy the key to it, so you can save it
+# somewhere else as a backup.
+system_info_encryption_local:
+  enabled: false # mandatory
+  localhost_key_dir: "{{ inventory_dir }}/system_encryption_keys" # mandatory
+  key_dir: /etc/scylla/system_encryption_keys # mandatory
+  key_name: system_key # mandatory
+  cipher_algorithm: AES # mandatory
+  secret_key_strength: 128 # mandatory
+  secret_key_block_mode: CBC # mandatory if you're not providing your own key
+  secret_key_padding: PKCS5 # mandatory if you're not providing your own key
+  complete_cipher_algorithm: "AES/CBC/PKCS5Padding" # mandatory. This is the value that is going to be set as cipher_algorithm in system_info_encryption, in scylla.yaml. Make sure you set it correctly.
+
+# KMIPKeyProvider:
+# All the variables under system_info_encryption_kmip are mandatory
+# Make sure you set kmip_hosts accordingly
+system_info_encryption_kmip:
+  enabled: false
+  cipher_algorithm: AES
+  secret_key_strength: 128
+  kmip_host: yourkmipServerIP.com
+
```

Comment:
And what about "Replicated Key Provider"?

Reply:
Replicated Key Provider is not supported for **system encryption**.
It's supported for **table encryption**, but enabling/disabling table encryption is not part of this PR.